docs/application/dotnet/guides/nui/linear-layout.md

```diff
@@ -0,0 +1,140 @@
+# Linear Layout
+
+[Overview](#overview)<br>
+[Orientation](#orientation)<br>
+[Alignment](#alignment)<br>
+[CellPadding](#cellpadding)<br>
+[Weight](#weight)<br>
+
+
+<a name="overview"></a>
+## Overview
+
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/linearTotal.png">
+</td>
+</tr>
+</table>
```

Comment:
Displaying the figure without an overview paragraph is awkward. 

How about moving the image line 22?
I mean the better position of the image is between "`LinearLayout` is a linear box layout ..." sentence and "Positioning children in a linear form,..." sentence.

Reply:
Okay. Moved the image to the below you mentioned.